plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -342,7 +342,7 @@ func ClusterRoles() []rbac.ClusterRole {
 				// this is for leaderlease access
 				// TODO: scope this to the kube-system namespace
 				rbac.NewRule("create").Groups(legacyGroup).Resources("endpoints").RuleOrDie(),
-				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-scheduler").RuleOrDie(),
+				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-default-scheduler").RuleOrDie(),
```

Comment:
He'd have to plumb through the lock through to a command-line arg and default it.  But we should also then update the multi-scheduler docs around input options and RBAC rules.